select/sdks/ios/releases.md

```diff
@@ -1,3 +1,17 @@
+# iOS SDK v2 Releases
+
+## 2.0.0
+- Removed card scanning so `Fidel.autoScan` has been removed.
```

Comment:
We should also add that we dropped support for Objective-C.

Reply:
@valllllll2000 please make sure to update the Android changelog as well with the changes from the doc I linked above.

Reply:
Done